src/main/resources/fabric.fluids.mixin.json

```diff
@@ -0,0 +1,14 @@
+{
+  "required": true,
+  "package": "io.github.fablabsmc.fablabs.api.fluidvolume.v1.mixin",
```

Comment:
All mixins should be moved to `io.github.fablabsmc.fablabs.fluidvolume.mixin`

Reply:
why's that?

Reply:
You usually don't have mixins within an api package

Reply:
I don't think the `api` in that verbose and unnecessarily long namespace literally means "api package", I thought it just meant "this is an api by fablabs"

Reply:
`api` means this is `api`. mixins are not `api`

Reply:
then why is fluid volume after it, and also there is an api package

Reply:
Once the mixin package is resolved, it will be fine to me.

In Fabric API, we follow this rule, and all other fablabs projects are expected to follow the same rule as well.

So move mixins to `io.github.fablabsmc.fablabs.fluidvolume.mixin` to be compliant.